packages/snaps-controllers/src/services/offscreen/OffscreenExecutionService.test.ts

```diff
@@ -126,7 +126,9 @@ describe('OffscreenExecutionService', () => {
       frameUrl: FRAME_URL,
     });
 
-    const hasDocument = getMockedFunction(chrome.offscreen.hasDocument);
+    const hasDocument = getMockedFunction(
+      chrome.offscreen.hasDocument,
+    ) as unknown as jest.MockedFunction<() => Promise<boolean>>;
```

Comment:
The purpose of `getMockedFunction` is to not require these type casts. If we need the type casts anyway, you might as well do:

```suggestion
    const hasDocument = chrome.offscreen.hasDocument as jest.MockedFunction<() => Promise<boolean>>;
```

Reply:
Even better if this works:

```suggestion
    const hasDocument = chrome.offscreen.hasDocument as jest.MockedFunction<typeof chrome.offscreen.hasDocument>;
```